app/javascript/packs/application.js

```diff
@@ -0,0 +1,10 @@
+/* eslint no-console:0 */
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+//
+// To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
+// layout file, like app/views/layouts/application.html.erb
+
+console.log('Hello World from Webpacker')
```

Comment:
console.log is a great debugging tool, however, once you know you are in react, you should remove this.

Reply:
This file comes with `webpacker` I don't know what it is for, so it can probably be deleted. It currently has nothing to do with what is rendered through react.

Reply:
![screen shot 2017-10-16 at 4 27 37 pm](https://user-images.githubusercontent.com/31625704/31633633-0516b54a-b28f-11e7-9bcb-1563adb22e83.png)
This is a screenshot of the product of deleting the application.js file.

Reply:
You still need that file, but just delete the console.log. You should also write a test to make sure the text is rendering on the page.